Test.hs

```diff
@@ -2,16 +2,15 @@
 
 module Main where
 
+import Data.Maybe
+
 import Data.ByteString.Char8 (pack)
 import qualified Data.ByteString                  as BS
 import qualified Pangraph                         as P
 import qualified Pangraph.GraphML.Parser          as GraphML_P
 import qualified Pangraph.VHDL.EnvironmentWriter  as VHDL_E
```

Comment:
There seem to be no need to give different qualified names to `VHDL` modules, so use `VHDL` for both.